src/vs/platform/terminal/node/terminalEnvironment.ts

```diff
@@ -72,3 +75,175 @@ export async function findExecutable(command: string, cwd?: string, paths?: stri
 	const fullPath = path.join(cwd, command);
 	return await exists(fullPath) ? fullPath : undefined;
 }
+
+export interface IShellIntegrationInjection {
+	/**
+	 * A new set of arguments to use.
+	 */
+	newArgs: string[] | undefined;
+	/**
+	 * An optional environment to mixing to the real environment.
+	 */
+	envMixin?: IProcessEnvironment;
+	/**
+	 * An optional array of files to copy from `source` to `dest`.
+	 */
+	copyFiles?: {
```

Comment:
`filesToCopy` instead?

`copyFiles` sounds to me like it would be of type `boolean`